### PR TITLE
Update japan.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the article with examples for other plans as it could be ambigious

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9925

File: microsoft-365/admin/support/japan.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ ROBOTS: NOINDEX, NOFOLLOW
 ## Phone number
 0120 996 680 (Office 365 Enterprise)
 
-0120 628 860 (Other Plans)
+0120 628 860 (Other Plans) (Example : Microsoft 365 Business Basic, Standard, Premium etc)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/support/japan.md]:
```suggestion
0120 628 860 (Other plans, such as Microsoft 365 Business Basic, Standard, Premium, and others)
```

File: microsoft-365/admin/support/japan.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Japanese: 24 hours a day, 7 days a week
 
 English: 24 hours a day, 7 days a week
 
-**Other Plans:**
+**Other Plans:** (Example : Microsoft 365 Business Basic, Standard, Premium etc):
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/support/japan.md]:
```suggestion
**Other plans** (such as Microsoft 365 Business Basic, Standard, Premium, and others):
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thaks

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder

**Comment by cmcatee-MSFT** [comment]
@msbemba & @yogkumgit The japan.md article was retired and replaced with [Find Microsoft 365 for business phone support numbers by country or region](https://learn.microsoft.com/en-us/microsoft-365/admin/support-contact-info?view=o365-worldwide). I've updated the info for Japan in the new article and will republish it shortly.

**Pull request closed without merging**